pkg/webservice/dao/allocation_ask_info.go

```diff
@@ -0,0 +1,42 @@
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package dao
+
+type AllocationAskLogDAOInfo struct {
+	Message   string `json:"message"`
+	Timestamp int64  `json:"timestamp"`
+	Count     int32  `json:"count"`
+}
+
+type AllocationAskDAOInfo struct {
+	AllocationKey      string                    `json:"allocationKey"`
+	AllocationTags     map[string]string         `json:"allocationTags"`
+	RequestTime        int64                     `json:"requestTime"`
+	ResourcePerAlloc   map[string]int64          `json:"resource"`
+	PendingCount       int32                     `json:"pendingCount"`
+	Priority           string                    `json:"priority"`
+	QueueName          string                    `json:"queueName"`
+	RequiredNodeID     *string                   `json:"requiredNodeId"`
```

Comment:
Why handle this one different than other optional values in objects?
A simple empty string would suffice, unless we go back and do this for all optional values.

Reply:
Fixed.